drivers/crypto/Kconfig.nrf_ecb

```diff
@@ -3,16 +3,12 @@
 # Copyright (c) 2020 Nordic Semiconductor ASA
 # SPDX-License-Identifier: Apache-2.0
 
+# Workaround for not being able to have commas in macro arguments
+DT_COMPAT_NORDIC_NRF_ECB := nordic,nrf-ecb
 
-menuconfig CRYPTO_NRF_ECB
+config CRYPTO_NRF_ECB
 	bool "nRF AES electronic codebook mode encryption"
-	depends on HAS_HW_NRF_ECB && !BT_CTLR
+	depends on $(dt_compat_enabled,$(DT_COMPAT_NORDIC_NRF_ECB))
+	depends on !BT_CTLR
```

Comment:
@anangl  should we have a comment explaining why !BT_CTRL is a dependency?

Reply:
I know it was un-commented, already, but perhaps we could add now the clarification.

Reply:
@ioannisg I added a note that explains this dependency. Please check if it is acceptable.

Reply:
Looks fine, thanks